test/framework/service.go

```diff
@@ -71,6 +71,35 @@ func (f *Framework) CreateOrUpdateServiceAndWaitUntilReady(ctx context.Context,
 	return finalizerFn, nil
 }
 
+func (f *Framework) CreateOrUpdateService(ctx context.Context, namespace string, service *v1.Service) (FinalizerFn, error) {
```

Comment:
I guess that we don't need this function anymore?

Reply:
Yep! missed it somehow. Also went ahead and reverted back the change to `basicQueryWorking`.